bubblewrap.c

```diff
@@ -1020,17 +1020,20 @@ setup_newroot (bool unshare_pid,
           /* There are a bunch of weird old subdirs of /proc that could potentially be
              problematic (for instance /proc/sysrq-trigger lets you shut down the machine
              if you have write access). We should not have access to these as a non-privileged
-             user, but lets cover them anyway just to make sure */
-          const char *cover_proc_dirs[] = { "sys", "sysrq-trigger", "irq", "bus" };
-          for (i = 0; i < N_ELEMENTS (cover_proc_dirs); i++)
+             user, but lets cover them anyway if we have no CAP_SYS_ADMIN just to make sure */
+          if (!opt_cap_add_or_drop_used || !(requested_caps[0] & CAP_TO_MASK_0 (CAP_SYS_ADMIN)))
```

Comment:
Hmm.  We don't need to cover them if they're *already* read-only right?  Would that help the logic here?

Reply:
I've tried this approach in the fixup patch I've added.  It should work as well:

previously:
```
$ bwrap --bind / / --proc /proc findmnt -U --noheadings -o TARGET -R /proc
/proc
├─/proc/sys
│ └─/proc/sys/fs/binfmt_misc
├─/proc/sysrq-trigger
├─/proc/irq
└─/proc/bus
```

with the PR:
```
$ ./bwrap --bind / / --proc /proc findmnt -U --noheadings -o TARGET -R /proc
/proc
└─/proc/sys/fs/binfmt_misc
```